src/components/suggestions.ts

```diff
@@ -57,19 +70,37 @@ export const addSuggestion = async (
   );
 };
 
+export const updateSuggestionState = async (ids: number[], state = SuggestionState.Pending): Promise<void> => {
+  const db = await openDB();
+  // Update created suggestions to pending suggestions in DB via ids
+  ids.map(async function (id) {
+    await db.run(
+      `
+      UPDATE suggestions
+      SET state = ?
+      WHERE id = ?;
+      `,
+      [suggestionStatesReadable[state], id]
```

Comment:
I missed this before, but why do we store readable strings in the database? The readable stuff should be for UI only, I thought we were always storing the lowercase key for state

Reply:
I think its easier to just store the readable stuff in the DB - I dont see why we shouldn't.

Reply:
I don't think this is easier, when a command handler receives a state param, it's parsed into the lowercase key, which is an enum value. If the database has readable strings, then you have to parse that parameter into a readable string. This means you are parsing stuff twice. Also, it's very bad practice to couple UI things with database stuff. What if we want to change the readable string into something else? That means we would have to do a DB migration to update all the rows.

Reply:
what you are doing here is already more complicated, if we store the lowercase key, we only need to pass in `state`, here you are passing in `suggestionStatesReadable[state]`

Reply:
Let's say if we want to do some DB operation on a particular state, we could just pass in something like `SuggestionState.SomeState`, instead of `suggestionStatesReadable[SuggestionState.SomeState]`